src/components/Dropdown/Dropdown.js

```diff
@@ -260,3 +261,5 @@ export default class Dropdown extends PureComponent {
     return dropdown;
   }
 }
+
+export default (!breakingChangesX ? Dropdown : null);
```

Comment:
When our feature flags are replaced with their default value, how would this code be compiled when it's set to `false`? Would it properly be stripped through uglify/terser? Just curious 👀

Reply:
Good question, it'll be `null`-export only as long as the code is tree-shakable. Added some code to demonstrate that in our Rollup build.